pkg/kubectl/cmd/expose.go

```diff
@@ -175,6 +176,9 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 		}
 		params["labels"] = kubectl.MakeLabels(labels)
 	}
+	if cmdutil.GetFlagBool(cmd, "headless") {
```

Comment:
I think for your changes here running `hack/update-generated-docs.sh` is enough.